web/main.ts

```diff
@@ -4,7 +4,9 @@
 /// <reference lib="dom.asynciterable" />
 /// <reference lib="deno.ns" />
 
-import "std/dotenv/load.ts";
+import { load } from "std/dotenv/mod.ts";
+
+const env = await load({ export: true });
```

Comment:
Yeah, in that case we might be able to do something like `Deno.env.set("APP_KEY", combinedEnv["WHATEVER_NAME"] || crypto.randomUUID())`. It might be better if we can find a way to derive a deterministic random string rather than using randomUUID if env is not given, but no big deal to be honest.